config/locales/es-MX.yml

```diff
@@ -4,6 +4,8 @@ es-MX:
     dashboard_welcome:
       welcome: "Bienvenido a Active Admin. Esta es la página de inicio predeterminada."
       call_to_action: "Para agregar secciones edite 'app/admin/dashboard.rb'"
+    admin_user:
+      details: "Detalles de administración"
```

Comment:
I think it'd be better "Detalles de administrador", since it refers to the admin user you are creating.